sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormField.cs

```diff
@@ -39,14 +39,22 @@ internal FormField(string name, FieldValue_internal fieldValue, IReadOnlyList<Re
             Name = name;
             LabelData = null;
 
-            IReadOnlyList<FormElement> FormElement = fieldValue.Elements != null
-                ? ConvertTextReferences(fieldValue.Elements, readResults)
-                : new List<FormElement>();
+            // Bounding box, page and text are not returned by the service in two scenarios:
+            //   - When this field is global and not associated with a specific page (e.g. ReceiptType).
+            //   - When this field is a collection, such as a list or dictionary.
+            //
+            // In these scenarios we do not set a ValueData.
+
+            if (fieldValue.BoundingBox.Count != 0 || fieldValue.Page != null || fieldValue.Text != null)
+            {
+                IReadOnlyList<FormElement> FormElement = ConvertTextReferences(fieldValue.Elements, readResults);
 
-            // TODO: FormEnum<T> ?
-            BoundingBox boundingBox = fieldValue.BoundingBox == null ? default : new BoundingBox(fieldValue.BoundingBox);
+                // TODO: FormEnum<T> ?
```

Comment:
not related to this PR, but do you know what `TODO: FormEnum<T>` means?

Reply:
No clue. It's likely to be related to an old version of the API, probably before the first preview. @annelo-msft may know. If no one knows what this means, I think it's safe to assume that it's not applicable anymore.